index.html

```diff
@@ -102,7 +102,7 @@ <h4>Profesional Art Printing Data More</h4>
                         <li>CSS</li>
                         <li>Javascript</li>
                     </ul>
-                    <button type="button">See Project</button>
+                    <button type="button" id="seeProject-2">See Project</button>
```

Comment:
- [ ] I have realized that the projects section is still hardcoded. Note that according to the project requirements, you are supposed to display the project lists dynamically by JavaScript. This generally involves having the details of each project in an array of objects and looping when displaying them to the screen when the page loads. Kindly display the projects lists dynamically so as to meet all the project requirements.

![image](https://user-images.githubusercontent.com/86318284/217452947-c87cce61-fffe-4e74-8fbf-4231a25fdb8a.png)